lib/elements/element.py

```diff
@@ -154,6 +155,16 @@ def get_multiple_int_param(self, param, default="0"):
             return [int(default)]
         return params
 
+    def get_json_param(self, param, default=None):
+        json_value = self.get_param(param, "{}")
+        try:
+            return json.loads(json_value, object_hook=DotDict)
+        except json.JSONDecodeError:
+            if default is None:
+                return {}
```

Comment:
Should this return a `DotDict` instance instead by default? Maybe should also only allow `DotDict` instances for default, too?

Reply:
Hmmm, interesting idea, I'll give it some thought.

Reply:
A consumer that assumes this will be a `DotDict` might crash out if it tries to manipulate the `dict` it gets in case of a default like a `DotDict`. At least, that's what I was thinking.